sql_generators/mobile_kpi_support_metrics/templates/union.view.sql

```diff
@@ -26,6 +26,11 @@ CREATE OR REPLACE VIEW
     CAST(NULL AS {{ field.type }}) AS {{ field.name }},
   {% endif %}
 {% endfor %}
+{% if 'adjust_network'.exists %}
+  `moz-fx-data-shared-prod.udf.organic_vs_paid_mobile`(adjust_network) AS paid_vs_organic,
+{% else %}
+  'Organic' AS paid_vs_organic,
```

Comment:
Does this mean other products like focus_ios which does not have any attribution data coming to us should always be classified as `Organic` here?

Reply:
Correct